zipline/utils/calendars/trading_calendar.py

```diff
@@ -704,6 +710,22 @@ def minute_to_session_label(self, dt, direction="next"):
 
         return current_or_next_session
 
+    def minute_index_to_session_labels(self, index):
+        """
+        Given a DatetimeIndex of market minutes, return a DatetimeIndex
+        of the corresponding session labels.
+
+        The market minutes must be in order.
```

Comment:
can you put numpy-style docstrings with Parameters, Returns, etc sections? (like the other public-facing methods in this class)